packages_rs/nextclade-web/src/helpers/formatQCCoverage.ts

```diff
@@ -0,0 +1,16 @@
+import type { QcResultCoverage } from 'src/algorithms/types'
+import type { TFunctionInterface } from 'src/helpers/TFunctionInterface'
+import { QcStatus } from 'src/algorithms/types'
+
+export function formatQCCoverage<TFunction extends TFunctionInterface>(t: TFunction, qcCoverage?: QcResultCoverage) {
+  if (!qcCoverage || qcCoverage.status === QcStatus.good) {
+    return undefined
+  }
+
+  const { score, coverage } = qcCoverage
+
+  return t('Coverage is low: {{coverage}}. QC score: {{score}}', {
+    coverage,
```

Comment:
Maybe express as `95%` rather than `0.95`?

Reply:
Sure, why not.